sdk/openai/openai/src/rest/outputModels.ts

```diff
@@ -650,12 +747,8 @@ export interface BatchImageGenerationOperationResponseOutput {
   expires?: number;
   /** The result of the operation if the operation succeeded. */
   result?: ImageGenerationsOutput;
-  /**
-   * The status of the operation
-   *
-   * Possible values: "notRunning", "running", "succeeded", "canceled", "failed"
-   */
-  status: string;
+  /** The status of the operation */
+  status: AzureOpenAIOperationStateOutput;
```

Comment:
Why are we still generating this batch image generation operation? Sounds like a bug in code gen.

Reply:
The TypeSpec still has definitions for batch image generations [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/routes/images.tsp#L45C1-L60C3). Let me check why it's not removed yet.

Reply:
It is tagged as removed, so I don't think code gen should generate it.

Reply:
From what I understand, the generated code has to support all API versions, so these models and operations are still emitted. It sounds like Python has a decorator to say which API version supports this operation, but I'm not sure if we have that in JS.

Reply:
Also, could we delete this model and any other models related to this operation?

Reply:
No, I don't think we default to last supported API version. For now, deleting all things related to this operation sounds like the right thing to do.